tests/test_auth.py

```diff
@@ -29,26 +29,29 @@ def test_get_whoami_no_credentials():
     whoami route returns a 400 error when no credentials are sent.
```

Comment:
```suggestion
    whoami route returns a 401 error when no credentials are sent.
```
And the next docstrings for which the test assertion has been updated with a `401` code error